src/arbcalls/arb_poly.jl

```diff
@@ -0,0 +1,287 @@
+###
+### **arb_poly.h** -- polynomials over the real numbers
+###
+
+### Types, macros and constants
+
+### Memory management
+arbcall"void arb_poly_init(arb_poly_t poly)"
+arbcall"void arb_poly_clear(arb_poly_t poly)"
+arbcall"void arb_poly_fit_length(arb_poly_t poly, slong len)"
+arbcall"void _arb_poly_set_length(arb_poly_t poly, slong len)"
+arbcall"void _arb_poly_normalise(arb_poly_t poly)"
+arbcall"slong arb_poly_allocated_bytes(const arb_poly_t x)"
+
+### Basic manipulation
+arbcall"slong arb_poly_length(const arb_poly_t poly)"
```

Comment:
This needs an `import Base: length` at the top

Reply:
There will be a similar function in `acb_poly.jl` as well. I'm still thinking about how to handle this in a good way.